src/numerics/petsc_matrix.C

```diff
@@ -470,6 +470,20 @@ void PetscMatrix<T>::update_preallocation_and_zero ()
   libmesh_not_implemented();
 }
 
+template <typename T>
+void PetscMatrix<T>::reset_preallocation()
+{
+#if !PETSC_VERSION_LESS_THAN(3,8,0)
```

Comment:
Oh I see.

 #if !PETSC_VERSION_LESS_THAN(3,9,0)

8-->9

I was thinking about 9. Why I gave you 8